controls/radspreadsheet/howto/customize-row-column-headers.md

```diff
@@ -62,8 +63,34 @@ That's it. The column heading is changed.
         
 
 >tipYou can download a runnable project of the previous example from our online SDK repository [here](https://github.com/telerik/xaml-sdk), the example is listed as __Spreadsheet / CustomRowAndColumnHeadings__.
+
+## States of the Row and Column Headings
+
+The headings of the rows and columns have different states. You can use these states to apply different styles of the items depending on whether they appear in a selection. This section will describe the possible states of the headings in RadSpreadsheet.
+
+
+* **HeadingState**: It is a property of the RowColumnHeadingBase type. It is an enum type and could have the following values:
+	* **Normal**: The heading is not included in any selection.
+	
+	* **Selected**: A cell from the row/column appears in a selection. 
+	
+	* **FullySelected**: All the cells included in the row/column are selected. 
+
           
+* **SelectAllControlState**: This is a property of the SelectAllControl and determines whether this control is selected. It is an enum type and could have the following values:
```

Comment:
Current Version:  This is a property of the SelectAllControl and determines whether this control is selected. It is an enum type and could have the following values:

Suggestion: A property of the SelectAllControl which is of type [the type]. The property gets or sets a value indicating (or determines) whether the control is selected. It is an enum and it could have the following values: